larray/session.py

```diff
@@ -190,6 +192,8 @@ def _dump(self, key, value, *args, **kwargs):
         self.handle[key] = value.dump(*args, **kwargs)
 
     def save(self):
+        if self.new_file:
+            self.handle['Sheet1'].delete()
```

Comment:
No, it is not possible to open a Workbook without a sheet. I updated XLWingsHandler to rename the first sheet when I first call the _dump() method.